tests/e2e-pw/global-teardown.ts

```diff
@@ -0,0 +1,3 @@
+module.exports = async () => {
+	console.log( 'Global teardown' ); // eslint-disable-line no-console
```

Comment:
Is this log necessary? What do you think about leaving the function empty instead? We will add to it later I expect, so we should keep it around to reduce the workload of someone who needs to use it, but the log does not add anything useful imo.

Reply:
Initially, I added it to prevent the `@typescript-eslint/no-empty-function` error. However, you have a good point. The log is not necessary and does not add any value here. 

With https://github.com/woocommerce/woocommerce-blocks/pull/9034#discussion_r1166698623 in mind, I went ahead and changed the file as follows:

https://github.com/woocommerce/woocommerce-blocks/blob/0a44da45a0c0130e1c7df827848cac6596c2e83c/tests/e2e-pw/global-teardown.ts#L1-L3